packages/blade/src/components/FileUpload/FileUpload.web.tsx

```diff
@@ -59,6 +66,24 @@ const _FileUpload: React.ForwardRefRenderFunction<BladeElementRef, FileUploadPro
   },
   ref,
 ): React.ReactElement => {
+  // eslint-disable-next-line @typescript-eslint/no-unnecessary-type-assertion
+  const { actionButtonText, dropAreaText } = rest as FileUploadVariableSizeProps;
+  const isSizeVariable = size === 'variable';
+  // 'actionButtonText' and 'dropAreaText' are only valid when size is 'variable'
+  if (__DEV__ && !isSizeVariable && (actionButtonText || dropAreaText)) {
```

Comment:
```suggestion
  if (__DEV__ && !isSizeVariable && (actionButtonText || dropAreaText)) {
```

I think you might have to created nested if like this-
```
if (__DEV__) {
  if (!isSizeVariable && (actionButtonText || dropAreaText)) {}
}
```

Since we use bundler to remove these __DEV__ checks in prod bundle, I am not sure if they'll remove this piece correctly otherwise

Reply:
Done